### PR TITLE
qbh limit and plots code

### PR DESCRIPTION
**Pull request by jingyucms**

### CODE REVIEW

File: chi_analysis/add_systematics_qbh.py [hunk 1]
```diff
@@ -19,48 +19,188 @@
...
+    fastNloPlot3=h3.Clone("fastNloQCD#chi4800_13000")
+    fastNloPlot3=fastNloPlot3.Rebin(len(chi_binnings[2])-1,"fastNloQCD#chi4800_13000_rebin1",chi_binnings[2])
+    fastNloFinalPlots=[fastNloPlot1,fastNloPlot2,fastNloPlot3]
+    fastNloFinalPlotsBackup=[]   
 
-massbins=[[4200,4800],[4800,13000]]
-slopes=[0.04,0.05]
+if isRS1QBH:
+    crossSectionMap=[[3500,0.4288],[4000,0.1464],[4500,0.05148],[5000,0.01829],[5500,0.006472],[6000,0.002250],[6500,0.0007599],[7000,0.0002461]]
+    outname="datacard_shapelimit13TeV_QBH_"+energy+"_RS1_chi_v1.root"
```
Comment (ahinzmann) [inline chi_analysis/add_systematics_qbh.py:44]: In order to able to quickly recalculate limits (and that happens often), can you give the QBHADD and QBHRS1 models different file names?

File: chi_analysis/add_systematics_qbh.py [hunk 2]
```diff
@@ -19,48 +19,188 @@
...
+    qbhfile=TFile.Open("root://eoscms//eos/cms/store/group/dpg_trigger/comm_trigger/L1Trigger/Jingyu/samples/DiJet/pythia_qbh_"+energy+"_RS1_v1.root")
+    print "root://eoscms//eos/cms/store/group/dpg_trigger/comm_trigger/L1Trigger/Jingyu/samples/DiJet/pythia_qbh_"+energy+"_RS1_v1.root"
+    signalHistFileName="qbh_"+str(energy)+"_RS1_chi_v1.root"
+    massbins=[[3600,4200],[4200,4800],[4800,13000]]
```
Comment (ahinzmann) [inline chi_analysis/add_systematics_qbh.py]: This is duplication of the code above. Rather only put stuff that is different in the if statement and everything else outside.

File: chi_analysis/add_systematics_qbh.py [hunk 3]
```diff
@@ -162,17 +271,18 @@
 print plots
         
 for massbin in massbins:            
-    plots[massbins.index(massbin)]=plots[massbins.index(massbin)].Rebin(len(chi_binnings[massbins.index(massbin)])-1,'QCDQBH'+energy+'_6#chi'+str(massbin[0])+'_'+str(massbin[1])+"_rebin1",chi_binnings[massbins.index(massbin)])
+    plots[massbins.index(massbin)]=plots[massbins.index(massbin)].Rebin(len(chi_binnings[massbins.index(massbin)])-1,'qbh_'+energy+'_6_#chi'+str(massbin[0])+'_'+str(massbin[1])+"_rebin1",chi_binnings[massbins.index(massbin)])
```
Comment (ahinzmann) [inline chi_analysis/add_systematics_qbh.py]: Can you name it QBHADD and QBHRS1?
Then the calculcate limit script will be able to calculate it in one go with all others and the calculate_limit_shape_qbh script can be dropped.

File: chi_analysis/calculate_shapelimit_qbh.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
+import os, sys
```
Comment (ahinzmann) [inline chi_analysis/calculate_shapelimit_qbh.py]: This script can actually be dropped. The calculate_shapelimit_13TeV script can also calculate the QBH in one go.

File: chi_analysis/plot_shapelimit_qbh.py [hunk 5]
```diff
@@ -0,0 +1,150 @@
+from ROOT import *
```
Comment (ahinzmann) [inline chi_analysis/plot_shapelimit_qbh.py]: This script can also be dropped, since plot_shapelimit_13TeV can calculate the QBH as well.

**Comment by jingyucms** [comment]
@ahinzmann Now the code is more compact and QBH histogram names changed to the convention that limits can be calculated by calculate_limit_shape_13TeV.py in one go.

**Pull request merged**